site/en/docs/lighthouse/performance/unminified-javascript/index.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Minify JavaScript
+description: |
+  Learn about the unminified-javascript audit.
+date: 2019-05-02
+updated: 2020-06-20
+web_lighthouse:
+  - unminified-javascript
+---
+
+Minifying JavaScript files can reduce payload sizes and script parse time.
+The Opportunities section of your Lighthouse report lists
+all unminified JavaScript files,
+along with the potential savings in [kibibytes (KiB)](https://en.wikipedia.org/wiki/Kibibyte)
+when these files are minified:
+
+<figure>
+  {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/aHumzRfDrBcuplUDCnvf.png", alt="A screenshot of the Lighthouse Minify JavaScript audit", width="800", height="212" %}
+</figure>
+
+## How to minify your JavaScript files
+
+Minification is the process of removing whitespace and any code that is not necessary
+to create a smaller but perfectly valid code file.
+[Terser](https://github.com/terser-js/terser) is a popular JavaScript compression tool.
+webpack v4 includes a plugin for this library by default to create minified build files.
+
+## Stack-specific guidance
+
+### Drupal
+
+Ensure you have enabled **Aggregate JavaScript files** in the **Administration**
+
+> **Configuration** > **Development** page. You can also configure more advanced
```

Comment:
This markup has been mangled during the transition: https://deploy-preview-2124--developer-chrome-com.netlify.app/docs/lighthouse/performance/unminified-javascript/#stack-specific-guidance

Reply:
This has been resolved in all instances of the article (English, Spanish, Portugese, and Korean). Turns out an encoded `>` character was interpreted as a blockquote.